docs/configuration.md

```diff
@@ -2326,7 +2326,7 @@ from this directory.
 # Inheriting Hadoop Cluster Configuration
 
 If you plan to read and write from HDFS using Spark, there are two Hadoop configuration files that
-should be included on Spark's classpath:
+should be included on Spark's class path:
```

Comment:
nit: everywhere in the documentation `classpath` is being used so changing just one instance will make the doc inconsistent. Lets keep this as it is.